Healthy/Classes/Utilities/Authentication/AuthenticationProtocol.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+public struct AuthenticatedUser {}
```

Comment:
- Move authenticated user to a separate file 
- Add the needed properties as agreed (ID - Provider Type - Name - Email)